Readme.md

```diff
@@ -24,14 +25,14 @@ scalable and reliable applications. This library intends to make it easy to impl
 - Easy to configure and use
 - Decorator-based API, just decorate and play
 - Different cache strategies out-of-the-box
-- Support for multiple storage backends ([In-memory](#in-memory), [Redis](#redis), [DiskCache](diskcache))
```

Comment:
> [In-memory](#in-memory)

Is there any hidden purposes behind this syntax? As I know, it is only for URLs.\
I think using **bold**, or _italic_, or "quotes" is enough.